package.json

```diff
@@ -30,8 +30,8 @@
   ],
   "author": "Nathan Black <nathanbl@uber.com>",
   "engines": {
-    "node": "14.19.3",
-    "npm": "6.14.17"
+    "node": "20.13.1",
+    "npm": "20.13.1"
```

Comment:
This does not look right. node and npm versions are generally not same. Current latest npm version is 10.8.1